src/wifes_calib.py

```diff
@@ -204,7 +204,7 @@ def extract_wifes_stdstar(cube_fn,
     
     # MZ: Commented out
     if x_ctr == None or y_ctr == None:
-        cube_im = numpy.sum(obj_cube_data, axis=0)
+        cube_im = numpy.nansum(obj_cube_data, axis=0)
```

Comment:
Ok with this for now, but I'll add an issue to investigate why we get NaNs, and maybe add some other error handling for it.

Reply:
See https://github.com/PyWiFeS/pipeline/issues/27